draft-01/post.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://skystandards.hns.siasky.net/draft-01/post.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "A representation of a post",
+    "type": "object",
+    "required": [
+        "id",
+        "content"
```

Comment:
> "repostOf": {
>             "$ref": "../draft-01/postReference.schema.json",
>             "description": "Full ID of the post being reposted (If this key is present, this post is a repost and does not need to contain a \"content\")"
>         }

Are we sure that `content` is required? What if `repostOf ` is present?

Reply:
Thanks, removed it!